test/controllers/password_resets_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
```

Comment:
Thanks for adding tests.  I would suggest a testing coverage utility like CodeClimate but you can try whichever you want.

e.g., https://stackshare.io/stackups/code-climate-vs-pullreview-vs-sonarqube